README.md

```diff
@@ -17,7 +17,10 @@ This is my first Website project with Microverse. And this is version 1 of the m
    HTML and CSS
 
 **Live Demo**
-Coming Soon!
+
+here you can see my first live demo of the site
+
+https://david63011.github.io/
 
 **Getting Started**
 
```

Comment:
- [ ] OPTIONAL
Kindly ensure that your ReadMe follows this [recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md) by Microverse. This will make your ReadMe appear more professional